library/src/main/java/com/devbrackets/android/exomedia/core/EMListenerMux.java

```diff
@@ -111,7 +118,15 @@ public void onError(EMExoPlayer emExoPlayer, Exception e) {
     }
 
     @Override
-    public void onStateChanged(boolean playWhenReady, int playbackState) {
+    public void onStateChanged(boolean playWhenReady, @PlaybackState int playbackState) {
```

Comment:
This is only capturing state changes from the ExoPlayer, we need to also account for MediaPlayer state changes